main.py

```diff
@@ -1,10 +1,59 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_data = json.load(file)
+        for nickname, value in players_data.items():
+
+            # create race
+            for _ in value["race"]:
+                race_name = value["race"]["name"]
+                race_description = value["race"]["description"]
+                if not Race.objects.filter(name=race_name).exists():
+                    Race.objects.create(
+                        name=race_name,
+                        description=race_description
+                    )
+                race_id = Race.objects.get(name=race_name).id
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create
Fix in all places